uart_example/README.md

```diff
@@ -0,0 +1,29 @@
+# flutter reactive BLE UART demo app
+
+This demo app shows an usecase for using flutter_reactive_ble library for building a UART emulation over BLE.
+
+This demo app uses one service: the Nordic UART Service with vendor-specific UUID:
+
+```
+Uuid _UART_UUID = Uuid.parse("6E400001-B5A3-F393-E0A9-E50E24DCCA9E");
+```
+
+The aforementioned service uses two characteristics: one for transmitting and one for receiving (as seen from the peer).
+
+```
+Uuid _UART_RX   = Uuid.parse("6E400002-B5A3-F393-E0A9-E50E24DCCA9E");
+Uuid _UART_TX   = Uuid.parse("6E400003-B5A3-F393-E0A9-E50E24DCCA9E");
+```
+
+## Usage
+
+This app can be used if the BLE peripheral implements the aforementioned services. The user interface is shown below:
+
+![alt text](./screenshot.png "User interface")
```

Comment:
"alt text" is not a useful alt text and "screenshot.png" is only useful if it's the only screenshot here (which it probably is, so fine).

Consider replacing "alt text" with something like "Application screenshot".